html5validator/tests/test_config.py

```diff
@@ -1,173 +1,187 @@
-# -*- coding: utf-8 -*-

```

Comment:
Why is this entire file different?

Same for all the files in `tests/config_files`?